worldguard-core/src/main/java/com/sk89q/worldguard/domains/registry/InvalidDomainFormat.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * WorldGuard, a suite of tools for Minecraft
+ * Copyright (C) sk89q <http://www.sk89q.com>
+ * Copyright (C) WorldGuard team and contributors
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.sk89q.worldguard.domains.registry;
+
+public class InvalidDomainFormat extends Exception {
```

Comment:
`InvalidDomainFormat` -> `InvalidDomainFormatException`

Reply:
Disagree, InvalidDomainFormat has the same format as InvalidFlagFormat
https://github.com/EngineHub/WorldGuard/blob/master/worldguard-core/src/main/java/com/sk89q/worldguard/protection/flags/InvalidFlagFormat.java#L22

Reply:
unfortunately this naming dates back to 2011 and the original addition of region flags. technically a breaking change to rename it now though. maybe can deprecate and migrate it (easily via extension) for the future if we want.

Reply:
I renamed InvalidFlagFormat to InvalidFlagFormatException with full backwards compatibility.
And InvalidDomainFormat is now InvalidDomainFormatException.